src/app/main/login/login.component.ts

```diff
@@ -3,7 +3,8 @@
 import { Router } from '@angular/router';
 import { AuthService } from 'src/app/services/auth.service';
 import {ApiService} from '../../services/api.service'
-
+import { HttpParams } from '@angular/common/http';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ticodacytest/chatApp1/pullRequest?prid=7784558)
```suggestion
import { HttpParams } from "@angular/common/http";
```